prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -693,29 +693,51 @@ def run_retrieval(  # type:ignore
     retrieval_type: str,
     metadata: dict[str, Any],
 ) -> tuple[str, str]:
+    context: str = ""
     prompt = output[PSKeys.PROMPTX]
     if retrieval_type == PSKeys.SUBQUESTION:
```

Comment:
Any reason why if else is not used?

Reply:
If-else for the retrieval type?  If any other retrieval types are added, this will have the logic specific.